frontend/pages/DashboardPage/cards/ActivityFeed/ActivityFeed.tsx

```diff
@@ -1,13 +1,14 @@
-import React, { useState } from "react";
+import React, { useRef, useState } from "react";
 import { useQuery } from "react-query";
 import { isEmpty } from "lodash";
 
 import activitiesAPI, {
   IActivitiesResponse,
 } from "services/entities/activities";
 
-import { IActivity } from "interfaces/activity";
+import { IActivity, IActivityDetails } from "interfaces/activity";
 
+import ShowQueryModal from "pages/schedule/ManageSchedulePage/components/ShowQueryModal/ShowQueryModal";
```

Comment:
When do we want to move components being used on multiple pages to global component directory? After 2 instances? 3?